README.md

```diff
@@ -14,6 +14,14 @@ $ npm install
 
 ### Getting started
 
+Before launching, the file at /app/assets/config/local.js must export an object containing the API's address and a valid Mapbox token, e.g.
+```
+module.exports = {
+  api: 'http://localhost:4000',
+  mbToken: 'pk.eyJ1I...'
```

Comment:
Since this is in the `production.js` which is committed, we can remove it from here